Containerfile

```diff
@@ -24,12 +24,16 @@ RUN apt-get clean && \
 RUN apt-get autoremove
 RUN apt-get install npm -y 
 
+RUN mkdir -pv /var/log/gunicorn/
+RUN mkdir -pv /var/run/gunicorn/
+
 COPY . /app/
 
 WORKDIR /app/
 
 RUN pip install -r requirements.txt
 RUN mv settings_local.templ.py settings_local.py
-RUN python manage.py makemigrations  
+RUN python3 manage.py makemigrations
+  
 
-CMD ["sh", "-c", "python3 manage.py migrate && python3 manage.py runserver 0.0.0.0:8000"]
+CMD ["sh", "-c", "python3 manage.py migrate && gunicorn -c config/gunicorn/dev.py"]
```

Comment:
The migrate command should be considered a prerequisite step that runs prior to the app running. As a matter of principle, the entry command should also represent the running of the app, and not any prerequisite steps in _building_ the app. I would recommend moving the `migrate` command up as a `RUN` directive, and leaving `gunicorn` as the only command on this line.